docs/repositories/files.md

```diff
@@ -25,34 +25,23 @@ Use the search box to filter the list and find specific files:
 
 ## File details
 
-Click a specific file to see more detailed analysis information for that file, including statistics on:
+Click a specific file to see more detailed analysis information for that file.
 
--   **Size:** Lines of code, source lines of code, and commented lines of code
--   **Structure:** Number of methods and ratio of source lines of code per number of methods
--   **Complexity:** Complexity and complexity per method
--   **Duplication:** Number of clones and duplicated lines of code
-
-The button **Ignore File** allows you to ignore the selected file on future Codacy analysis.
+The header of the file detail page displays the same code quality metrics as the Files page, along with an **Ignore file** link to ignore the file, and a link to view the file on your Git provider.
 
 ![File detail](images/files-details.png)
 
 Depending on the available analysis information for the file, Codacy displays one or more of the following tabs:
```

Comment:
Are we dropping **Complexity/Method** metric? 🤔 
I noticed we don't have this value on the header, and we also don't have a Complexity tab...

Reply:
I believe all of these are being dropped:

- Lines of code
- Source lines of code
- Commented lines of code
- Number of Methods
- sLoC / Method
- Complexity / Method
- Duplicated lines of code